Dropdown/Dropdown.js

```diff
@@ -167,33 +243,40 @@ const DropdownBase = kind({
 		skin: PropTypes.string,
 
 		/**
-		 * The primary title text of Dropdown.
+		 * Text displayed in the Dropdown when nothing is selected.
 		 *
 		 * The title will be replaced with the selected item, if set.
 		 *
 		 * @type {String}
 		 * @public
 		 */
-		title: PropTypes.string
+		title: PropTypes.string,
+
+		/**
+		 * Width of the Dropdown.
+		 *
+		 * @type {('smallest'|'small'|'medium'|'large'|'x-large'|'huge')}
+		 * @default 'medium'
+		 * @public
+		 */
+		width: PropTypes.oneOf(['smallest', 'small', 'medium', 'large', 'x-large', 'huge'])
```

Comment:
Sandstone has `tiny` instead of `smallest`. Is there a reason for this diverge?

Reply:
I wanted to keep agate's sizing names. Button and Icon have smallest, small, large, huge. Which do you think is the best naming?

Reply:
I don't know why we used `smallest` in the first place, `tiny` seems better. But I guess then we should change all the proptypes related to this. Let's keep it as is for now.